src/utils/exr-parser.worker.js

```diff
@@ -17,102 +17,103 @@
  *  };
  */
 
-let openEXRLoaded = false;
-let queuedJobs = [];
-let OpenEXR;
+exports.__esModule = true;
 
-self.addEventListener('message', (event) => {
+var EXR = require("../exr-wrap/exr-wrap.js");
+
+// eslint-disable-next-line no-restricted-globals
+var ctx = self;
+var openEXRLoaded = false;
+var queuedJobs = [];
+var openEXR;
+
+ctx.addEventListener('message', function (event) {
     if (!openEXRLoaded) {
         queuedJobs.push(event.data);
-    } else {
+    }
+    else {
         handleJob(event.data);
     }
 });
 
-
-const exrwrapPath = require('file-loader?name=exr-wrap.js!../exr-wrap/exr-wrap.js');
-const exrwrapWasmPath = require('file-loader?name=exr-wrap.wasm!../exr-wrap/exr-wrap.wasm');
```

Comment:
This part doesn't just work when compiling and importing this as an npm module, this load the wasm from the file-loader, but then the exr-wrap.js will load it in a web worker env from the browser and still attempt to resolve as an XMLHttpRequest. So I realize the intent here is to get Webpack to copy the wasm file to the built output, but that's already happening in the package.json scripts.